hack/jenkins/linux_integration_tests_kvm.sh

```diff
@@ -26,7 +26,7 @@
 set -e
```

Comment:
You should remove the entries from the `set-pending` script also

Reply:
Good catch. Done.